src/main/java/com/three_tech_solutions/slot_app/data/mappers/StudentMapper.java

```diff
@@ -10,7 +10,8 @@ public class StudentMapper {
         public static StudentResponse toResponse(Student student) {
             return new StudentResponse(
                     student.getName(),
-                    student.getLastname()
+                    student.getLastname(),
+                    student.isEnabled() //se asigna al campo isActive del response
```

Comment:
Este comentario no hace falta


```suggestion
                    student.isEnabled()
```